server/comms/test: noresponse send error

Go CI 1.13 test failure.

log:
```
 ok  	decred.org/dcrdex/server/coinlock	1.788s
--- FAIL: TestOnline (0.92s)
    comms_test.go:708: noresponse send error: write tcp 127.0.0.1:44728->127.0.0.1:33797: write: connection reset by peer
FAIL
FAIL	decred.org/dcrdex/server/comms	1.203s
ok  	decred.org/dcrdex/server/db	1.746s
```

@chappjc comment:
> It looks like the pingPeriod might be too fast or the pongWait too short vs pingPeriod in comms_test.go for the CI machines. 90 ms is a terribly fast ping period so I wouldn't be surprised if these pathetic CI machines just freeze for longer than that for no apparent reason. Otherwise I don't know why the conn write would have gotten connection reset by peer there.

If memory serves correctly, all of these sleep/wait related issues happen in 1.13. No hard evidence that this is causal, just an observation.

--- a/client/comms/wsconn_test.go
+++ b/client/comms/wsconn_test.go
@@ -75,7 +75,7 @@ func TestWsConn(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	pingWait := time.Millisecond * 200
+	pingWait := time.Second
 
 	type conn struct {
 		sync.WaitGroup

--- a/server/comms/comms_test.go
+++ b/server/comms/comms_test.go
@@ -610,7 +610,7 @@ func TestOnline(t *testing.T) {
 
 	keyPath := filepath.Join(tempDir, "rpc.key")
 	certPath := filepath.Join(tempDir, "rpc.cert")
-	pongWait = 100 * time.Millisecond
+	pongWait = time.Second
 	pingPeriod = (pongWait * 9) / 10
 	server, err := NewServer(&RPCConfig{
 		ListenAddrs: []string{":0"},

--- a/server/comms/comms_test.go
+++ b/server/comms/comms_test.go
@@ -610,7 +610,7 @@ func TestOnline(t *testing.T) {
 
 	keyPath := filepath.Join(tempDir, "rpc.key")
 	certPath := filepath.Join(tempDir, "rpc.cert")
-	pongWait = time.Second
+	pongWait = time.Millisecond * 500
 	pingPeriod = (pongWait * 9) / 10
 	server, err := NewServer(&RPCConfig{
 		ListenAddrs: []string{":0"},

--- a/server/comms/comms_test.go
+++ b/server/comms/comms_test.go
@@ -704,8 +704,8 @@ func TestOnline(t *testing.T) {
 		return err
 	}
 
-	// Sleep for a few  pongs to make sure the client doesn't disconnect.
-	time.Sleep(pongWait * 3)
+	// Sleep for a couple of pongs to make sure the client doesn't disconnect.
+	time.Sleep(pongWait * 2)
 
 	// Positive path.
 	err = sendToDEX("ok", "{}")

